CONTRIBUTORS

```diff
@@ -17,23 +21,21 @@ Thank you!
     Alexander Lukyanov <lav@yar.ru>
     Alexandre Chappaz <alexandrechappaz@gmail.com>
     Alexandre SIMON <alexandre.simon@ciril.fr>
-    Alex Dowad <alexinbeijing@gmail.com>
     Alexey Veselovsky <alexey.veselovsky@eykontech.com>
     Alexis Robert <alexis.robert@gmail.com>
-    Alex Rousskov <rousskov@measurement-factory.com>
-    Alex Rousskov <rousskov@squid-cache.org>
-    Alex Wu <alex_wu2012@hotmail.com>
     Alin Nastac <mrness@gentoo.org>
     Alter <alter@alter.org.ua>
+    Ambrose Li <ambrose.li@gmail.com>
+    Amish <3330468+amishmm@users.noreply.github.com>
+    Amish <amishxda@users.noreply.github.com>
     Amit Klein <amit.klein@safebreach.com>
-    Amos Jeffries
     Amos Jeffries <amosjeffries@squid-cache.org>
     Amos Jeffries <squid3@treenet.co.nz>
     Amos Jeffries <yadij@users.noreply.github.com>
-    Amos <squid3@treenet.co.nz>
```

Comment:
@yadij (or anybody else reading this), if you want to remove or edit some of your own CONTRIBUTORS entries, this branch is a good opportunity to do so. You can edit the in-branch copy of that file (even using GitHub file editing!) at any time before this PR is merged. Thank you.

P.S. Please do not misinterpret my comment as implying that you must remove or change some of your entries! Leaving all of them "as is" is fine.